wrappers/react-native/android/build.gradle

```diff
@@ -34,6 +34,17 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')
```

Comment:
```
> Task :mattrglobal_pairing-crypto-rn:processReleaseManifest
package="com.mattrglobalpairingcryptorn" found in source AndroidManifest.xml: /Users/ken.huang/code/js-credential-sdk/examples/wallet-sdk-react-native-example/node_modules/@mattrglobal/pairing-crypto-rn/android/src/main/AndroidManifest.xml.
Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported, and the value is ignored.
Recommendation: remove package="com.mattrglobalpairingcryptorn" from the source AndroidManifest.xml: /Users/ken.huang/code/js-credential-sdk/examples/wallet-sdk-react-native-example/node_modules/@mattrglobal/pairing-crypto-rn/android/src/main/AndroidManifest.xml.
```

manually tested on RN 0.73 upgrade, added this section into deps fixed android build issue.